qtconsole/mainwindow.py

```diff
@@ -109,6 +113,17 @@ def set_window_title(self):
         if ok:
             self.setWindowTitle(title)
 
+    def create_tab_with_existing_kernel(self):
+        name, ok = QtGui.QInputDialog.getText(self,
+                                               "Connect to Existing Kernel",
+                                               "Kernel name:")
+        if not ok:
+            return
+        connection_file = os.path.join(jupyter_runtime_dir(), name)
+        widget = self.connection_frontend_factory(connection_file)
+        name = 'bla'
```

Comment:
As I said in the pull request mesaage - I consider tab naming an open issue. So I figured I should get some input before writing throw-away code.